.gitignore

```diff
@@ -55,4 +55,5 @@ networkx.egg-info/
 .pytest_cache
 
 # Virtual environment directory
-networkx-dev/
\ No newline at end of file
+networkx-dev/
+venv/
```

Comment:
This PR shouldn't require changes to .gitignore - if you want to name your development environment differently that's totally fine, but it shouldn't be included in the changes in the PR.